remappings.txt

```diff
@@ -11,5 +11,6 @@
 
 @script/=script/
 @contracts/=src/contracts/
-@test/=test/
+@interfaces/=src/interfaces/
```

Comment:
Do we need to include this in the template? If so can you do that please?

Reply:
```
remote: Permission to open-dollar/contracts-template.git denied to daopunk.
fatal: unable to access 'https://github.com/open-dollar/contracts-template.git/': The requested URL returned error: 403
```